src/SFA.DAS.Assessor.Functions/Functions/Ofqual/OfqualDataReader.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using CsvHelper;
+using Microsoft.Azure.WebJobs;
+using SFA.DAS.Assessor.Functions.Domain.Entities.Ofqual;
+using SFA.DAS.Assessor.Functions.Domain.OfqualImport.Interfaces;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+namespace SFA.DAS.Assessor.Functions.Functions.Ofqual
+{
+    public class OfqualDataReader
+    {
+        private readonly IOfqualDownloadsBlobFileTransferClient _blobFileTransferClient;
+
+        public OfqualDataReader(IOfqualDownloadsBlobFileTransferClient blobFileTransferClient)
+        {
+            _blobFileTransferClient = blobFileTransferClient;
+        }
+
+        [FunctionName(nameof(ReadOrganisationsData))]
+        public async Task<IEnumerable<OfqualOrganisation>> ReadOrganisationsData([ActivityTrigger] string filePath, ILogger logger)
+        {
+            if (!await FileExists(filePath))
+            {
+                logger.LogError($"Failed to find Organisations data file at {filePath}.");
+                return new List<OfqualOrganisation>();
```

Comment:
I see you have changed the Qualifications one below

Reply:
Drat, fixed now